summit/auxtel-vent-gates/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+---
+version: '3.6'
+
+services:
+  vent_controller:
+    image: lsstts/vents-controller-aarch64:${CYCLE}
```

Comment:
Your images have the revision so a `${rev}` needs to be added to the end of this line.